services/venice-controller/src/main/java/com/linkedin/venice/controller/server/StoresRoutes.java

```diff
@@ -1000,8 +1000,8 @@ public void internalHandle(Request request, StoreHealthAuditResponse veniceRespo
         String store = request.queryParams(NAME);
         Map<String, RegionPushDetails> details = admin.listStorePushInfo(cluster, store);
 
-        veniceResponse.setStoreName(store);
-        veniceResponse.setClusterName(cluster);
+        veniceResponse.setName(store);
```

Comment:
`setStoreName` is still easier to understand. It'd be great to rename `setName` to `setStoreName`. Intellij should be able to do it at once easily but feel free to drop this comment.

Reply:
Adam, thanks for the review! Yes, I agree that setStoreName would be more intuitive. However, since ControllerResponse.name is in the base class and it is the base of our API response, and I am not sure if renaming it could cause any dependency issues e.g. breaking their build, so probably I will not do it in a clean up change :-)